examples/postgres-operator/pgadmin/values.yaml

```diff
@@ -11,4 +11,4 @@ resources:
     memory: "512Mi"
 env:
   email: "zarf@example.local"
-  password: "###ZARF_GIT_AUTH_PUSH###"
+  password: "###ZARF_VAR_PGADMIN_PASSWORD###"
```

Comment:
This isn't declared anywhere in a package.

Reply:
`git stash` strikes again.....